### ISSUE TITLE
can not tflite squeezedet graph by freeze_graph.py

### ISSUE THREAD

**Issue by Davari393** [tensorflow/tensorflow#32521]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 18.04):
- TensorFlow installed from (binary):
- TensorFlow version (1.15.0-dev20190707):

Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: ADD, ARG_MAX, CAST, CONCATENATION, CONV_2D, DIV, EXP, FLOOR, GREATER, LOGISTIC, MAXIMUM, MAX_POOL_2D, MINIMUM, MUL, PACK, REDUCE_MAX, RESHAPE, SOFTMAX, STRIDED_SLICE, SUB, TRANSPOSE, UNPACK. Here is a list of operators for which you will need custom implementations: RandomUniform.

**Comment by haozha111** [comment]
Hi, there is one op 'RandomUniform' that is not supported by tflite as built-in op. TF Lite can also support ops from tensorflow. Could you follow instructions here (https://www.tensorflow.org/lite/guide/ops_select) ?

**Comment by tonytu123** [comment]
vfd

**Comment by haozha111** [comment]
Update:

we are adding `RandomUniform` as a built-in op. Jaeyoo, could you help follow up on this issue once your code is checked-in?

**Comment by jaeyoo** [comment]
Hi @Davari393 ,

I am sorry for the belated updates. As the op is not yet supported and the support in builtin ops are delayed, as @haozha111 suggested, the following conversion will work for now.

```python
converter = tf.lite.TFLiteConverter.from_concrete_functions([model])
converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS, tf.lite.OpsSet.SELECT_TF_OPS]
tflite_model = converter.convert()
```

**Comment by tensorflowbutler** [comment]
Hi There,

 We are checking to see if you still need help on this, as you are using an older version of tensorflow which is officially considered end of life . We recommend that you upgrade to the latest 2.x version and let us know if the issue still persists in newer versions. Please open a new issue for any help you need against 2.x, and we will get you the right help. 

 This issue will be closed automatically 7 days from now. If you still need help with this issue, please provide us with more information.
